docs/user-guide/python.md

```diff
@@ -44,64 +44,78 @@ or in a submission script) using:
+Installing packages to your local environment can now be done as follows.
 
-Sometimes, you may need several different versions of the same package installed, for example, due to dependency issues. Virtual environments allow you to manage these conflicting requirements. The first step is to run the commands contained in the above section so that you can install the `virtualenv` package which will manage your environments. To install `virtualenv` run:
-```
-pip install --user virtualenv  # The --user flag indicates this should be installed in the user's package folder
-```
-Next you must create a folder for the virtual environment's files to live in and tell `virtualenv` to set this folder up for storing virtual environment 'stuff'. This is done by running the command
-```
-mkdir /work/t01/t01/auser/<<name of your virtual environment>>  # Create the folder
-virtualenv -p /opt/cray/pe/python/3.8.5.0/bin/python /work/t01/t01/asuser/<<name of your virtual environment>>  # -p flag means use this python interpreter
-```
-Finally, you're ready to `activate` your environment. This is done by running
-```
-source /work/t01/t01/auser/<<name of your virtual environment>>/bin/activate
-```
-Once your environment is activated you will be able to install packages as usual using `pip install <<package name>>`. These packages will only be available within this environment. When running code that requires these packages you must activate the environment, by adding the above `source ... activate` line of code to any submission scripts.
+    (myvenv) auser@ln01:~> python -m pip install <package name>
+
+Running `pip` directly as in `pip install <package name>` will also work, but we show the `python -m` approach
+as this is consistent with the way the virtual environment was created.
+
+!!! tip
+    The ARCHER2 compute nodes cannot access the `/home` file system, which means you may need to run
+    `export XDG_CACHE_HOME=${HOME/home/work}` if you're working from within an interactive session as
+    that export command will ensure the pip cache is located off `/work`.
+
+And when you have finished installing packages, you can deactivate the environment by running the `deactivate` command.
+
+    (myvenv) auser@ln01:~> deactivate
+    auser@ln01:~> 
+
+The packages you have installed will only be available once the local environment has been activated. So, when running code that requires these packages,
+you must first activate the environment, by adding the activation command to the submission script, as shown below.
+
+=== "Full system"
+    ```
+    #!/bin/bash --login
+
+    #SBATCH --job-name=myvenv
+    #SBATCH --nodes=2
+    #SBATCH --ntasks-per-node=64
+    #SBATCH --cpus-per-task=2
+    #SBATCH --time=00:10:00
+    #SBATCH --account=[budget code]
+    #SBATCH --partition=standard
+    #SBATCH --qos=standard
+
+    source /work/t01/t01/auser/myvenv/bin/activate
+
+    srun python ${SLURM_SUBMIT_DIR}/myvenv-script.py
```

Comment:
Should this have the standard `--hint=nomultithread --distribution=block:block` as for other example scripts?